crmd/lrm.c

```diff
@@ -1823,6 +1832,10 @@ construct_op(lrm_state_t * lrm_state, xmlNode * rsc_op, const char *rsc_id, cons
 
     params = xml2list(rsc_op);
     g_hash_table_remove(params, CRM_META "_op_target_rc");
+    
+    if (!is_remote_lrmd_ra(NULL, NULL, rsc_id)) {
+        op->versioned_params = first_named_child(rsc_op, XML_TAG_VER_ATTRS);
```

Comment:
first_named_child() does not duplicate the data, so this is just a pointer into rsc_op (which may be freed later). I think you want to wrap this with copy_xml().